core/src/main/java/com/kin/ecosystem/core/data/blockchain/BlockchainSource.java

```diff
@@ -97,6 +97,13 @@ void signTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount,
 	void sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount, @NonNull String orderID,
 		@NonNull String offerID);
 
+	/**
+	 *
+	 * @param transactionEnvelope
+	 * @return
+	 */
+	String extractTransactionId(String transactionEnvelope);
```

Comment:
can you explain on the comments here how?

Reply:
done